src/org/aavso/tools/vstar/plugin/IPlugin.java

```diff
@@ -52,6 +52,11 @@ public interface IPlugin {
 	 */
 	abstract public String getDescription();
 
+	/**
+	 * Get a document file name for this plugin.
+	 */
```

Comment:
It would be worth adding that this document can be a full URL or a file name that assumes the plugin doc dir as  a base URL. This could help plugin developers.